docs/config.md

```diff
@@ -1,457 +1,147 @@
-
-- name: Copy extra_data.py to server
-  copy: src=baltimore_example/extra_data.py dest=/opt/piecewise/
-- pip: name=ipaddress state=latest
-
-# ------------------------------------------------------------------
-#
-# 4) Change the Ansible command that imports your geodata to reflect the 
-# name of the folder containing your shapefiles as well as the shapefile itself.
-#
-#    For example, change:
-#
-#        command: ogr2ogr -f PostgreSQL -t_srs EPSG:4326 -nln seattle_blkgrpce10 -nlt 
-#        MultiPolygon 'PG:user=postgres dbname=piecewise' 
-#        /opt/piecewise_web/seattle_blkgrpce10/CENSUS10_blkgrp_WGS.shp
-#    to:
-#        command: ogr2ogr -f PostgreSQL -t_srs EPSG:4326 -nln maryland_blkgrps -nlt 
-#        MultiPolygon 'PG:user=postgres dbname=piecewise' 
-#        /opt/piecewise_web/maryland_blkgrps/cb_2015_24_bg_500k.shp
-#
-- name: Ingest census blocks to postgres
-  command: ogr2ogr -f PostgreSQL -t_srs EPSG:4326 -nln seattle_blkgrpce10 -nlt MultiPolygon 'PG:user=postgres dbname=piecewise' /opt/piecewise_web/seattle_blkgrpce10/CENSUS10_blkgrp_WGS.shp
-
-# ------------------------------------------------------------------
-#
-# 5) Rename the folder name "seattle_example" to match your folder name. 
-#
-#    For example, change:
-#       copy: src=seattle_example/piecewise_config.json dest=/etc/piecewise/config.json
-#    to:  
-#       copy: src=baltimore_example/piecewise_config.json dest=/etc/piecewise/config.json
-
-- name: Install piecewise configuration
-  copy: src=baltimore_example/piecewise_config.json dest=/etc/piecewise/config.json
-
-# ------------------------------------------------------------------
-# The remaining Ansible commands do not need to be changed for customization of Piecewise. 
-# ------------------------------------------------------------------
-
-- name: Restart uwsgi so piecewise config is detected
-  service: name=uwsgi state=restarted
-- command: python extra_data.py chdir=/opt/piecewise
+Shapefiles are the most widely used geodata format used in the GIS community. The US Census Bureau provides downloadable shapefiles for a variety of boundaries in the US: [https://www.census.gov/geo/maps-data/data/cbf/cbf_blkgrp.html](https://www.census.gov/geo/maps-data/data/cbf/cbf_blkgrp.html). In the US, GIS people in cities or states often publish shapefiles for their communities that often amend or correct the shapefiles provided by the US Census Bureau. 
 
-```
+```TO DO: add reference to international shapefile sources```
 
-##### Edit the main Piecewise configuration file
+Once you locate the shapefile(s) you need, it's good practice to open the shapefile in [QGIS](http://www.qgis.org/en/site/) or another GIS program to confirm that it meets the requirements to use within your Piecewise application. When you open the shapefile(s) in your GIS program, **make a note of the name of its unique key field.** You'll use that later in these instructions.
```

Comment:
A few additional instructions for QGIS: 
(0) Download QGIS
(1) Project > New
(2) Layer > Add Layer > Add Vector Layer with local .shp files 
(3) Choose target granularity level. In the case of gadm.org shapefiles, 0 equals country (least specific) to 3 is most specific. 
(4) Remove unused local files. At this point, you should have 6 files: .cpg, .csv, .dbf, .prj, .shp, .shx. For more info on Shapefiles: http://wiki.gis.com/wiki/index.php/Shapefile